src/main.rs

```diff
@@ -1,11 +1,11 @@
 #![feature(array_chunks)]
 // Should be disabled when player inventory saving and loading is implemented
-#![feature(const_in_array_repeat_expressions)]
 
 /// Minecraft implementations.
 pub mod minecraft;
 /// Mojang API implementation.
 pub mod mojang;
+// Utilities
```

Comment:
What is this comment for?

Reply:
I accidentally left a comment over from my own modifications outside the scope of this project. This can be removed